spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'post_index', type: :feature do
+  user = User.first
+  post = user.posts.first
+  before(:each) do
+    visit user_posts_path(User.first.id)
+  end
+
```

Comment:
- [ ] Try to create a user and a post for that user and then add feature tests for the post_index page.

Reply:
Done ✔